libtransmission/net.cc

```diff
@@ -516,12 +516,16 @@ std::optional<tr_address> tr_address::from_string(std::string_view address_sv)
 std::string_view tr_address::display_name(char* out, size_t outlen) const
 {
     TR_ASSERT(is_valid());
+    if (!is_valid())
+    {
+        return "Invalid address"sv;
+    }
     return evutil_inet_ntop(tr_ip_protocol_to_af(type), &addr, out, outlen);
```

Comment:
I think it would be better to handle the null return value instead; address being invalid may not be the only case it returns null for. Moreover, we're expecting a valid address at this point (based on the above assertion) so checking for it is odd — should probably be either one or the other.

Reply:
eb33fa7 Good point.